Smooch.postman_collection.json

```
@@ -6289,6 +6189,84 @@
 									"body": "{\n    \"integration\": {\n        \"_id\": \"5735ddfd48011972d621dc0a\",\n        \"type\": \"wechat\",\n        \"appId\": \"c69175d6d125b772b\",\n        \"webhookSecret\": \"3889794ab2fd4a70940a97c4b4a6372e\"\n    }\n}"
 								}
 							]
+						},
+						{
+							"name": "WhatsApp",
+							"request": {
+								"method": "POST",
+								"header": [
+									{
+										"key": "Authorization",
+										"value": "Bearer {{JWT}}"
+									},
+									{
+										"key": "Content-Type",
+										"value": "application/json"
+									}
+								],
+								"body": {
+									"mode": "raw",
+									"raw": "{\n\t\"type\": \"whatsapp\",\n    \"deploymentId\": \"<whatsappDeploymentId>\"\n}"
```

You'd be disappointed if I did not nitpick you a bit, so here it is:

```suggestion
									"raw": "{\n    \"type\": \"whatsapp\",\n    \"deploymentId\": \"<whatsappDeploymentId>\"\n}"
```

---

:+1: mon ptit nitpickeux